app/graphql/types/api_user_type.rb

```diff
@@ -0,0 +1,6 @@
+module Types
+  class ApiUserType < GraphQL::Schema::Object
```

Comment:
Any reason this isn't a BaseObject?
```suggestion
   class ApiUserType < BaseObject
```